### PR TITLE
Links to learning platform

### PR DESCRIPTION
**Pull request by elise2106**
Links added to all lessons

### CODE REVIEW

File: _lessons/design-thinking/design-process.md [hunk 1]
```diff
@@ -17,3 +17,6 @@ Resources
 - [An Introduction to Design Thinking: Process Guide](https://dschool-old.stanford.edu/sandbox/groups/designresources/wiki/36873/attachments/74b3d/ModeGuideBOOTCAMP2010L.pdf) from Stanford's d.school
 - [The Double Diamond: Strategy + Execution of the Right Solution](https://www.thoughtworks.com/insights/blog/double-diamond) by Jonny Schneider at Thoughtworks
 - For something more in-depth, read [A study of the design process](https://www.designcouncil.org.uk/sites/default/files/asset/document/ElevenLessons_Design_Council%20(2).pdf) from the Design Council in the UK, which resulted in the double diamond model.
+
```
Comment (nevanscott) [inline _lessons/design-thinking/design-process.md:20]: I think this extra line is unnecessary.

File: _lessons/design-thinking/personas.md [hunk 2]
```diff
@@ -14,6 +14,15 @@ Learning Objectives
 - Develop a persona based on findings from user interviews
 
 
+Learning Objectives
```
Comment (nevanscott) [inline _lessons/design-thinking/personas.md]: Should this say "Resources"?

File: _lessons/design-thinking/usability-testing.md [hunk 3]
```diff
@@ -20,6 +20,12 @@ Resources
 - [How to Run an Unmoderated Remote Usability Test](https://www.youtube.com/watch?v=vthW7mmn854) from UX Mastery
 - [Usability Testing](https://www.usability.gov/how-to-and-tools/methods/usability-testing.html) at usability.gov
 
+- [Usability as a design framework](http://learn.ironhack.com/#/learning_unit/5040) from Ironhack Learning Platform
```
Comment (nevanscott) [inline _lessons/design-thinking/usability-testing.md]: The empty line above this is unnecessary.

File: _lessons/information-architecture/content-first.md [hunk 4]
```diff
@@ -2,3 +2,12 @@
 title: Content First
 class_date: Sept 13, 2018
 ---
+
+Resources
+----------
+
+- [Intro to information architecture](http://learn.ironhack.com/#/learning_unit/5023) from Ironhack Learning Platform
```
Comment (nevanscott) [inline _lessons/information-architecture/content-first.md]: Not sure all of these will belong here, but I think it's fine for now until I add more detail to these lessons.

File: _lessons/information-architecture/forms.md [hunk 5]
```diff
@@ -2,3 +2,11 @@
...
 class_date: Sept 19, 2018
 ---
+
+
+Resources
+---------
+
+- [Forms & icons](http://learn.ironhack.com/#/learning_unit/5072) from Ironhack Learning Platform
+- [Designing multistate screens](http://learn.ironhack.com/#/learning_unit/5075) from Ironhack Learning Platform
+- [Lighting, shadows and consistency](http://learn.ironhack.com/#/learning_unit/5076) from Ironhack Learning Platform
```
Comment (nevanscott) [inline _lessons/information-architecture/forms.md]: This one should go back with UI design (week 2 I think)

File: _lessons/interaction-design/hig.md [hunk 6]
```diff
@@ -27,3 +27,10 @@ The HIG is... detailed. You're not going to read through it all in one go. Here'
 [Navigation Bars]: https://developer.apple.com/design/human-interface-guidelines/ios/bars/navigation-bars/
 [Tab Bars]: https://developer.apple.com/design/human-interface-guidelines/ios/bars/tab-bars/
 [Toolbars]: https://developer.apple.com/design/human-interface-guidelines/ios/bars/toolbars/
+
+Resources
+--------------
+- [iOs vs Material design](http://learn.ironhack.com/#/learning_unit/5085) from Ironhack Learning Platform
```
Comment (nevanscott) [inline _lessons/interaction-design/hig.md]: Even if it's not this way in the learning platform, I'd like to use the conventional capitalisation for iOS.

File: _lessons/interaction-design/ui-design.md [hunk 7]
```diff
@@ -21,6 +21,10 @@ Resources
 - [7 Rules for Creating Gorgeous UI (Part 1)](https://medium.com/@erikdkennedy/7-rules-for-creating-gorgeous-ui-part-1-559d4e805cda) and [Part 2](https://medium.com/@erikdkennedy/7-rules-for-creating-gorgeous-ui-part-2-430de537ba96) by Erik Kennedy
 - [Dribbble](https://dribbble.com)
 - [Microcopy: Tiny Words With A Huge UX Impact](https://uxplanet.org/microcopy-tiny-words-with-a-huge-ux-impact-90140acc6e42) by Nick Babich
+- [User interface design principles](http://learn.ironhack.com/#/learning_unit/5033) from Ironhack Learning Platform
+- [User interface fundamentals](http://learn.ironhack.com/#/learning_unit/5060) from Ironhack Learning Platform
+- [Design systems](http://learn.ironhack.com/#/learning_unit/5079) from Ironhack Learning Platform
+- [Landing pages](http://learn.ironhack.com/#/learning_unit/5098) from Ironhack Learning Platform
```
Comment (nevanscott) [inline _lessons/interaction-design/ui-design.md]: Let's put this one over with Content First for now

File: _lessons/interaction-design/visual-design.md [hunk 8]
```diff
@@ -25,6 +25,14 @@ Resources
 - Explore colors with [Dribbble color search](https://dribbble.com/colors/)
 - Explore typographic layouts at [Typewolf](https://www.typewolf.com)
 
+- [Intro to interaction design](http://learn.ironhack.com/#/learning_unit/5031) from Ironhack Learning Platform
```
Comment (nevanscott) [inline _lessons/interaction-design/visual-design.md]: I think there's an extra blank line above this.

File: _lessons/design-thinking/design-process.md [hunk 9]
```diff
@@ -17,3 +17,6 @@ Resources
...
+- [Introduction to User Research](http://learn.ironhack.com/#/learning_unit/4992) from Ironhack Learning Platform
+- [Double diamond & Problem Statement](http://learn.ironhack.com/#/learning_unit/5014) from the Ironhack Learning Platform.
```
Comment (nevanscott) [inline _lessons/design-thinking/design-process.md]: This one has an extraneous "." at the end

**Review by nevanscott — CHANGES_REQUESTED** [review]
Thanks for taking a first pass at this! I left a few notes below.

Overall, I'm wondering what you think about using a header called "Ironhack Resources" or "Ironhack Learning Platform Resources" or something, instead of always say from the Ironhack Learning Platform? I think it might be easier to skim. @lrbloomfield @dzc1 What do you guys think?

**Comment by lrbloomfield** [comment]
Yes, I agree with suggestion of additional section. I'd title it "Ironhack Learning Platform Resources."

**Comment by nevanscott** [comment]
@lrbloomfield Elise went with "Ironhack Learning Platform" as the heading. I'm cool with it and don't want to block this further. :)

@elise2106 Great work, thanks for putting this together!

**Review by nevanscott — APPROVED** [review]
💯 Ready to merge, @elise2106 I'll let you do the honors!

**Pull request merged**